prompts.py

```diff
@@ -0,0 +1,59 @@
+orchestrator_prompt = """
+You are an AI assistant made for information analysis.\n
+Let's think step by step.\n
+Step 1: Look at the image provided.\n
+Step 2: Think about what is a pie chart, what is a bar chart, what is a line graph, what is a list before you respond.\n
+Step 3: Identify the type of chart in the image.\n
+Step 4: Check if what you think is correct.\n
+Step 5: Respond with "pie chart" if it is a pie chart, "bar chart" if it is a bar chart, "line graph" if it is a line graph, "list" if it is a list, "trash" if it is anything else.\n
+"""
+
+pie_chart_prompt =  """
+You are working at a firm and you will be given a pie chart. 
```

Comment:
Ja bih ovde pre dao intro u smislu "You are AI assistent and your task is to..."